kubetest2/kubetest2-gce/deployer/deployer_helpers.go

```diff
@@ -0,0 +1,26 @@
+package deployer
+
+import (
+	"fmt"
+	"os"
+)
+
+func (d *deployer) setKubernetesPathIfNotSet() error {
+	if d.kubernetesRootPath == "" {
+		path, err := os.Getwd()
+		if err != nil {
+			return fmt.Errorf("Failed to get current working directory for setting Kubernetes root path: %s", err)
+		}
+		d.kubernetesRootPath = path
+	}
+
+	return nil
+}
+
+// verifyBuildFlags only checks flags that are needed for Build
+func (d *deployer) verifyBuildFlags() error {
+	if err := d.setKubernetesPathIfNotSet(); err != nil {
+		return err
```

Comment:
Shouldn't

return  d.setKubernetesPathIfNotSet()

be enough.

but d.setKubernetesPathIfNotSet will change d.kubernetesRootPath
is this expected?

Reply:
Done. This is expected. I could, instead, have the flag default to `os.Getwd()` but that would include error-handling logic in the flag setup. I'm open to suggestions.